test/run_e2e_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# This script sources the built ROS2 workspace, installs the necessary Python dependencies,
+# and executes the python end to end tests for the system_metrics_collector package.
+
+ROS_WS_DIR=ros_ws/install/
```

Comment:
Is there a way to invoke this through `colcon test` instead of an external mechanism?

Reply:
I'm not sure, but also unclear why we would need to run via ```colcon test```.

Reply:
General reasoning is that that's how tests are executed in a ROS2 package

Reply:
Emerson, we want to avoid that because the goal is to share this shell script between canaries (i.e. getting the binary through apt), and local tests.

Reply:
`ros2 pkg prefix system_metrics_collector` ?

Reply:
If we want to use ```colcon test``` to execute the e2e test (intially specified as quality control for this repo via GH Actions), then I think this should be implemented as a separate PR. Seems like it could be a good first issue for an interested contributor, otherwise we're needlessly blocking.